src/components/useraccount/UserProfile.vue

```diff
@@ -1,108 +1,27 @@
 <template>
-   <div v-if="me"
-        class="my-account-content">
+   <div :class="{ 'col-sm-9': showTab.showPersonalDetails, 'col-sm-12': showTab.showMyOrders }">
     <div class="row">
       <div class="col-sm-3">
         <!--{{> myaccount/my-account-sidebar myPersonalDetailsTab=true}}-->
-        <SidebarMenu/>
+        <SidebarMenu :showTab="showTab"/>
```

Comment:
to me personally, I don't think the name *showTab* is clear and it took me a while to understand what it is. do you think there is a clearer name for it?